packages/wallets/src/components/Base/WalletClipboard/WalletClipboard.tsx

```diff
@@ -16,7 +18,9 @@ const WalletClipboard = ({
     textCopy,
 }: TProps) => {
     const [, copy] = useCopyToClipboard();
+    const { isMobile } = useDevice();
     const [isCopied, setIsCopied] = useState(false);
+    const [isHovered, setIsHovered] = useState(false);
```

Comment:
https://usehooks-ts.com/react-hook/use-hover

Can you update this to use useHover instead? Best to use the utils in this hook - they are tried and tested and optimised